scripts/deploy.ts

```diff
@@ -0,0 +1,54 @@
+import { Wallet } from 'ethers';
+
+import {
+  Chains,
+  HyperlaneCore,
+  MultiProvider,
+  objMap,
+} from '@hyperlane-xyz/sdk';
+import { RouterConfig, chainConnectionConfigs } from '@hyperlane-xyz/sdk';
+
+import { CollateralConfig, SyntheticConfig } from '../src/config';
+import { HypERC721Deployer } from '../src/deploy';
+
+const connectionConfigs = {
+  celo: chainConnectionConfigs.celo,
+  ethereum: chainConnectionConfigs.ethereum,
+};
+
+async function deployNFTWrapper() {
+  console.info('Getting signer');
+  const signer = new Wallet(
+    'pkey',
+  );
+
+  const multiProvider = new MultiProvider(connectionConfigs);
+  multiProvider.rotateSigner(signer)
+  const core = HyperlaneCore.fromEnvironment('mainnet', multiProvider);
+  const config = {
+    celo: {
+      type: 'SYNTHETIC_URI',
+      name: 'Flow3rs 2022',
+      symbol: 'FLOW3',
+      totalSupply: 0,
+      owner: signer.address,
+      mailbox: '0x1d3aAC239538e6F1831C8708803e61A9EA299Eec',
```

Comment:
Should this come from the SDK?

Reply:
I think because we don't have v2 deployed on mainnet he did this